dlup/cli/mask.py

```diff
@@ -0,0 +1,197 @@
+# coding=utf-8
+# Copyright (c) dlup contributors
+"""CLI utilities to handle masks"""
+import argparse
+import json
+from multiprocessing import Pool
+from typing import cast
+
+import numpy as np
+from shapely.ops import unary_union
+from tqdm import tqdm
+
+from dlup._image import Resampling
+from dlup.annotations import AnnotationType, Polygon, WsiAnnotations, WsiSingleLabelAnnotation
+from dlup.cli import file_path
+from dlup.data.dataset import TiledROIsSlideImageDataset
+from dlup.experimental_backends import ImageBackend
+from dlup.tiling import TilingMode
+from dlup.utils.mask import mask_to_polygons
+from functools import partial
+
+
+def _get_sample(index, dataset, index_map, scaling):
+    output = {}
+    sample = dataset[index]
+    _mask = np.asarray(sample["image"])
+    for index in index_map:
+        curr_mask = (_mask == index).astype(np.uint8)
+        if curr_mask.sum() == 0:
```

Comment:
`not curr_mask.any()` may be faster

https://stackoverflow.com/questions/18395725/test-if-numpy-array-contains-only-zeros